README.md

```diff
@@ -15,35 +15,13 @@
 [![Dependencies status](https://badgen.net/david/dep/cerner/terra-dev-site)](https://david-dm.org/cerner/terra-dev-site)
 [![devDependencies status](https://badgen.net/david/dev/cerner/terra-dev-site)](https://david-dm.org/cerner/terra-dev-site?type=dev)
 
-Dynamically builds a site based on site configuration, navigation configuration and page configuration.
+A webpack plugin to dynamically builds a terra-application documentation and testing site. See [terra-ui](https://engineering.cerner.com/terra-ui/dev_tools/terra-dev-site/terra-dev-site/about) for documentation.
 
-The provided webpack plugin includes a call to the generateAppConfig script. The generateAppConfig script builds out static config to the ./dev-site-config/build folder. generateAppConfig also discovers pages based on it's configuration. After the static config has been built webpack continues to run, pulling in the static config, and producing the webpack bundle.
+## Installing the package
 
-Provides the following default configuration:
-
-* `config/site/site.config.js`
-* `config/site/navigation.config.js`
-* `config/webpack/webpack.config.js`
-
-* [Getting Started](#getting-started)
-* [Versioning](#versioning)
-* [Contributing](#contributing)
-* [LICENSE](#license)
-
-## Getting Started
-
-* Install with [npm](https://www.npmjs.com): `npm install --save-dev @cerner/terra-dev-site`
-
-## Peer Dependencies
-
-This component requires the following peer dependencies be installed in your app for the component to properly function.
-
-| Peer Dependency | Version |
-|-|-|
-| react | ^16.8.5 |
-| react-dom | ^16.8.5 |
-| terra-toolkit | ^5.2.0 |
-| webpack | ^4.28.1 |
+```bash
+npm install --save-dev @cerner/terra-dev-site
```

Comment:
Can we also list the peer dependencies to this quick copy & paste command?

Reply:
updated